target/linux/qualcommax/image/ipq807x.mk

```diff
@@ -332,6 +332,21 @@ define Device/redmi_ax6
 endef
 TARGET_DEVICES += redmi_ax6
 
+define Device/tplink_deco-x80-5g
+	$(call Device/FitImage)
+	$(call Device/UbiFit)
+	DEVICE_VENDOR := TP-Link
+	DEVICE_MODEL := deco-x80-5g
```

Comment:
This should be `Deco X80-5G`.

Reply:
Please stop marking stuff as resolved when it wasn't, and change to @musashino205 suggestion

Reply:
I did change it from `x80-5g` to `deco-x80-5g` sorry I'm a bit lost in what is strings and what are id's but I'm learning.